services/supervisor/src/lib.rs

```diff
@@ -29,19 +34,20 @@ use crate::{
         InstanceDescriptor, SUPERVISOR_INSTANCE_ID, SUPERVISOR_INSTANCE_NAME,
     },
 };
+use exonum_merkledb::Snapshot;
 
 mod api;
 mod errors;
 mod proto;
+mod proto_structures;
 mod schema;
 mod transactions;
 
 #[derive(Debug, ServiceFactory)]
 #[exonum(
-    crate = "crate",
-    proto_sources = "proto::schema",
+    proto_sources = "proto",
     artifact_name = "exonum-supervisor",
-    implements("transactions::Transactions")
+    implements("transactions::SupervisorInterface")
```

Comment:
Why not always use `Transactions` interface name for the service own transactions?

Reply:
[Try to agree between yourselves](https://github.com/exonum/exonum/pull/1473#discussion_r327947649)

Personally I don't care much about how it's called.

Reply:
It's `<service-name>Interface` in every example. So let's be consistent in either way